internal/apis/certmanager/v1alpha2/conversion.go

```diff
@@ -27,6 +29,9 @@ func Convert_v1alpha2_CertificateSpec_To_certmanager_CertificateSpec(in *Certifi
 		return err
 	}
 
+	out.URIs = *(*[]string)(unsafe.Pointer(&in.URISANs))
+	out.EmailAddresses = *(*[]string)(unsafe.Pointer(&in.EmailSANs))
```

Comment:
I changed the names of these fields for our internal types to match our public types.